Dockerfile

```diff
@@ -0,0 +1,61 @@
+# heavily borrowed from https://elixirforum.com/t/cannot-find-libtinfo-so-6-when-launching-elixir-app/24101/11?u=sigu
```

Comment:
could you update readme as well to reflect this new way to deploy to GCP (readme says gitlab). 
Either include info in readme or make a separate markdown file and reference in readme.
Include where to put customizable config into (eg secrets file and/or env variables and putting in new secret for phoenix)

Reply:
Let me include the instructions right here.